src/lycophron/project.py

```diff
@@ -72,10 +68,32 @@ def recreate_project(self):
 
         db.recreate_db()
 
-    def validate_project(self):
+    def _is_valid_url(self, config):
+        parsed_url = urlparse(config["ZENODO_URL"])
+        if not all([parsed_url.scheme, parsed_url.netloc]):
+            raise ValueError(f"Invalid URL provided: {config['ZENODO_URL']}.")
+
+    def _is_valid_token(self, config):
+        session = create_session(config["TOKEN"])
+        url = config["ZENODO_URL"] + "/api/me"
+        response = session.get(url)
+        if response.status_code != 200:
+            raise ValueError(
+                f"Invalid token or URL provided. Token: {config['TOKEN']}, URL: {config['ZENODO_URL']}"
+            )
+
+    def validate_project(self, config):
         # Configs exist
-        # Configs are valid
-        raise NotImplementedError("Project validation not implemented yet")
+        for required_value in required_configs:
+            if not config[required_value]:
+                raise ValueError(f"Missing required config: {required_value}")
+        # Configs are valid (sqlite uri is already validated, as the db needs to be initialized in order to reach this)
+        self._is_valid_url(config)
+        self._is_valid_token(config)
+        # Folder for files does exist
+        folder_name = "files"
+        if not os.path.exists(folder_name) or not os.path.isdir(folder_name):
+            raise ValueError(f"Missing required folder: {folder_name}")
```

Comment:
suggestion: we could use a custom error class for this. E.g:

```python
class InvalidDirectoryError(LycophronError):
    error_type = "DIRECTORY"
    hint = (
        "Create the appropriate directory structure. See documentation on how to do it."
    )
```
That way the [ErroHandler](https://github.com/plazi/lycophron/blob/main/src/lycophron/errors.py#L61) will log the exception and provide a hint to the user.